client/src/App.tsx

```diff
@@ -20,6 +23,10 @@ import './App.css';
 import NavBar from './components/NavBar/NavBar';
 import AllNotifications from './pages/AllNotifiactions/AllNotifications';
 
+const stripePromise = loadStripe(
+  'pk_test_51JYBH4HXzYVsCZt0Mls8k6b8UIXWlzgUTCinfBuNzmrqYLlUeUQuV4gcj7sePE1cDlUP2sRkFWJmAMgqovjZOEMd006aTSQqIg',
```

Comment:
This would better belong in an environment variable. https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
Thanks, Ethan. I was looking for that docs